<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MqttWindow</class>
 <widget class="QMainWindow" name="MqttWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MQTT Client</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>771</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>ClientId</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Host</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QPushButton" name="connectButton">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="4">
      <widget class="QLineEdit" name="usernameEdit"/>
     </item>
     <item row="2" column="1" colspan="4">
      <widget class="QLineEdit" name="passwordEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="4">
      <widget class="QLineEdit" name="clientIdEdit"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="hostEdit">
       <property name="text">
        <string>broker.hivemq.com</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Username</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QSpinBox" name="portSpin">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
       <property name="value">
        <number>1883</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>771</width>
      <height>103</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="2">
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" rowspan="2">
      <widget class="QTextEdit" name="textEdit"/>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_5"/>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Topic</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Text</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>771</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="1">
      <widget class="QTabWidget" name="logTab">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="msgReceivedTab">
        <attribute name="title">
         <string>MsgReceived</string>
        </attribute>
        <widget class="QTextBrowser" name="msgTextBrowser">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>769</width>
           <height>319</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="subscribeTab">
        <attribute name="title">
         <string>Subscribe</string>
        </attribute>
        <widget class="QListView" name="subscribeListView">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>0</y>
           <width>681</width>
           <height>291</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="addSubscribeButton">
         <property name="geometry">
          <rect>
           <x>700</x>
           <y>0</y>
           <width>61</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
        <widget class="QPushButton" name="deleteSubscribeButton">
         <property name="geometry">
          <rect>
           <x>700</x>
           <y>40</y>
           <width>61</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Log</string>
        </attribute>
        <widget class="QTextBrowser" name="logTextBrowser">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>0</y>
           <width>751</width>
           <height>291</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>789</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
